dnf-behave-tests/features/microdnf/install.feature

```diff
@@ -0,0 +1,86 @@
+@no_installroot
+Feature: microdnf install command on packages with xml:base
+
+
+Background: Using repositories dnf-ci-fedora and dnf-ci-thirdparty-updates
+  Given I delete file "/etc/yum.repos.d/*.repo" with globs
+
+
+@bz1734350
+Scenario: Install package from local repodata with local packages
+#1. local repo with local packages
+Given I use repository "dnf-ci-fedora"
+ When I execute microdnf with args "install kernel"
+ Then the exit code is 0
+  And stderr is empty
```

Comment:
Hmm, we should _definitely_ check the installation happened and the output of the command as well (in all the tests). There's a step for dnf to do that: `Transaction is following`, which checks both the dnf output and the rpm transaction that took place. We'll probably need to add a step like `microdnf transaction is following`, which will share the rpm transaction bit, but will parse microdnf output to verify it. Let me know if you can do that or you need some help with it.

Reply:
I agree and I need support to update "transaction.py" to match microdnf transactions.

Example:
```
[...]
    And Transaction is following                                         # features/steps/transaction.py:144 0.133s
      | Action  | Package                                  |
      | install | kernel-core-0:4.18.16-300.fc29.x86_64    |
      | install | kernel-modules-0:4.18.16-300.fc29.x86_64 |
      | install | kernel-0:4.18.16-300.fc29.x86_64         |
      Traceback (most recent call last):
        File "/usr/lib/python3.7/site-packages/behave/model.py", line 1329, in run
          match.run(runner.context)
        File "/usr/lib/python3.7/site-packages/behave/matchers.py", line 98, in run
          self.func(context, *args, **kwargs)
        File "features/steps/transaction.py", line 146, in then_Transaction_contains
          check_transaction(context, 'contains')
        File "features/steps/transaction.py", line 126, in check_transaction
          check_dnf_transaction(context, mode)
        File "features/steps/transaction.py", line 94, in check_dnf_transaction
          dnf_transaction = parse_transaction_table(lines)
        File "/opt/behave/features/steps/lib/dnf.py", line 119, in parse_transaction_table
          table_begin = find_transaction_table_begin(lines)
        File "/opt/behave/features/steps/lib/dnf.py", line 93, in find_transaction_table_begin
          raise RuntimeError("Transaction table start not found")
      RuntimeError: Transaction table start not found

Last Command: microdnf install kernel

Last Command stdout:
Package                                 Repository       Size
Installing:                                                  
 kernel-4.18.16-300.fc29.x86_64         dnf-ci-fedora  6.2 kB
 kernel-core-4.18.16-300.fc29.x86_64    dnf-ci-fedora 52.3 kB
 kernel-modules-4.18.16-300.fc29.x86_64 dnf-ci-fedora 48.2 kB
Transaction Summary:
 Installing:        3 packages
 Reinstalling:      0 packages
 Upgrading:         0 packages
 Removing:          0 packages
 Downgrading:       0 packages
Running transaction test...
Installing: kernel-core;4.18.16-300.fc29;x86_64;dnf-ci-fedora
Installing: kernel-modules;4.18.16-300.fc29;x86_64;dnf-ci-fedora
Installing: kernel;4.18.16-300.fc29;x86_64;dnf-ci-fedora
Complete.

```